packages/cli/src/commands/generate/page/__tests__/page.test.js

```diff
@@ -118,6 +118,13 @@ beforeAll(() => {
       pathName('/bazinga-ts/{id:Int}', 'typescript-param-with-type')
     ),
   })
+  jest.spyOn(console, 'info').mockImplementation(() => {})
```

Comment:
This file has pretty many tests. Do we need to mock console for all of them, or can we be more deliberate about what tests we mock for?

Reply:
I see that only a few tests actually call the handler which is where the output is generated. I've changed to only quiet the output for the tests which call the handler function.